flow/Arena.h

```diff
@@ -547,6 +547,14 @@ inline static Standalone<StringRef> makeString( int length ) {
 	return returnString;
 }
 
+inline static Standalone<StringRef> makeAlignedString( int alignment, int length ) {
```

Comment:
Well, that's fair.  I'll go fixup clang-format into the commits.

Reply:
Okay, so the tooling that I have to do this relies on the files already being clang formatted.  Which... isn't what I thought was going to happen.  I've burnt too much time today trying to get a clang-format merge driver that only clang formats the diffed lines, so I'm just going to do a giant single clang format patch at the end :/

Reply:
I thought `git clang-format` only formats the diffed lines.

Reply:
It does, and even if I were to push a clang format patch onto this PR, I'd still have to figure out what to do about all the conflicts it would generate for the patches that are the next 4 PRs that are already on top of this.  :/